R/data-ddl-utils.R

```diff
@@ -0,0 +1,99 @@
+#' Function runs the `code`, masks the `code` and creates `teal_data` object.
+#' @param data (`teal_data`) object
+#' @param code (`language`) code to evaluate
+#' @param input (`list`) containing inputs to be used in the `code`
+#' @param input_mask (`list`) containing inputs to be masked in the `code`
+#'
+#' @return `teal_data` object
+#'
+#' @export
+eval_and_mask <- function(data,
```

Comment:
this should belong to `teal.code` as a separate method or wrapper

Reply:
I consider `eval_and_mask` to be exception which should be avoided in standard workflow. I thought it would be better if we demonstrate `eval_and_mask` only in case of `ddl` because in contrary it might be used in other cases to break reproducibility.

Reply:
How this can be avoided for connectors with auth?

Reply:
Connectors will use `eval_and_mask`, but it will be developer by us, so we know it will be used in a right context.